datafusion/src/sql/planner.rs

```diff
@@ -2797,6 +2801,45 @@ mod tests {
         quick_test(sql, expected);
     }
 
+    #[test]
+    fn over_order_by_with_window_frame_double_end() {
+        let sql = "SELECT order_id, MAX(qty) OVER (ORDER BY order_id RANGE BETWEEN 3 PRECEDING and 3 FOLLOWING), MIN(qty) OVER (ORDER BY order_id DESC) from orders";
+        let expected = "\
+        Projection: #order_id, #MAX(qty), #MIN(qty)\
+        \n  WindowAggr: windowExpr=[[MAX(#qty)]] partitionBy=[]\
```

Comment:
I think it would be a good idea to have the window bounds appear in the explain plans